truss/remote/remote_cli.py

```diff
@@ -28,12 +28,18 @@ def inquire_remote_config() -> RemoteConfig:
         validate=NonEmptyValidator(),
     ).execute()
 
+    remote_inference_base_domain = inquirer.text(
```

Comment:
will this prompt the user to add a base domain every time? Most of the time they won't have one, so this might be somewhat confusing. I would leave this out for now, and then have people modify this via `truss configure` instead